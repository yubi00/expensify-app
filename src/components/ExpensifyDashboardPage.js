import React from 'react'
import ExpenseList from './ExpenseList'
import ExpenseListFilters from './ExpenseListFilters'
import ExpenseSummary from './ExpenseSummary'

const ExpensifyDashboardPage = () => (
    <div>
        <ExpenseSummary/>
        <ExpenseListFilters/>
        <ExpenseList/>
     </div>
)

export default ExpensifyDashboardPage