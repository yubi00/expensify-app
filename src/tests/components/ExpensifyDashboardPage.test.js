import React from 'react'
import { shallow } from 'enzyme'
import ExpensifyDashboardPage from '../../components/ExpensifyDashboardPage'

test('should render ExpensifyDashboardPage correctly', () => {
    const wrapper = shallow(<ExpensifyDashboardPage/>)
    expect(wrapper).toMatchSnapshot()
})