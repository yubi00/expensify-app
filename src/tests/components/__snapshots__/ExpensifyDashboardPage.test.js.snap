// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensifyDashboardPage correctly 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
